app/src/main/res/layout/fragment_offline_edit.xml

```diff
@@ -29,5 +29,7 @@
         android:background="@drawable/rounded_button"
         android:gravity="center"
         android:text="@string/txtSendAll"
+        android:paddingLeft="70dp"
+        android:paddingRight="80dp"
```

Comment:
In terms of padding, these should always be the same for left and right, and since we do support LTR languages, please include paddingEnd and paddingStart. I would recommend also reducing this to ensure that all languages are supported without having to make the button bigger should the text be longer once translated. Maybe 16dp, 20dp?